docs/src/features.md

```diff
@@ -1,14 +1,21 @@
 # Features
 
-This package enhances existing concepts from Sublime Text and introduces new concepts not native to Sublime Text. This page provides an overview of the most important capabilities. The capabilities/concepts are accessible in different ways. Some are accessible via the Context Menu by right-clicking with your mouse, or via the top Menu Bar. Others are part of an existing workflow. Almost all capabilities can also be bound to a key with a key binding.
+This package enhances existing concepts from Sublime Text and introduces new concepts not native to Sublime Text.
+This page provides an overview of the most important capabilities.
+The capabilities / concepts are accessible in different ways.
+Some are accessible via the Context Menu by right-clicking with your mouse, or via the top Menu Bar.
+Others are part of an existing workflow.
+Almost all capabilities can also be bound to a key with a [key binding](https://www.sublimetext.com/docs/key_bindings.html).
 
 ## Auto Complete
 
-[Example GIF 1](https://user-images.githubusercontent.com/6579999/128551586-d2ca51f7-6d16-4e12-bec5-ad2e6bc6c1f1.gif)
+![Example GIF 1](https://user-images.githubusercontent.com/6579999/128551586-d2ca51f7-6d16-4e12-bec5-ad2e6bc6c1f1.gif)
```

Comment:
Those were on purpose links so that the user can click it and start playing from the beginning instead of scrolling to a confusing animation that might be mid-way in the progress and makes the user wonder what the hell is happening.